Sources/PointFree/Admin/NewEpisodes.swift

```diff
@@ -13,7 +13,7 @@ import Prelude
 import Styleguide
 import Tuple
 
-let showNewEpisodeEmailMiddleware: AppMiddleware<Prelude.Unit> =
+let showNewEpisodeEmailMiddleware: M<Prelude.Unit> =
```

Comment:
do we want to use `M` instead of `AppMiddleware`?

Reply:
I didn't notice the overlap till I moved `M` and `MT` into TODO.swift. I figure this is all a temp situation till we rethink HttpPipeline, so cool with any names. Preference before merging?

Reply:
cool, let's keep as is then!